packages/manager/apps/dedicated/client/app/components/ip/agoraOrder/ip-ip-agoraOrder.html

```diff
@@ -1,215 +1,253 @@
+            data-id="offer"
+            data-name="offer_selection"
+            data-header="{{:: 'ip_agora_table_header_offer' | translate }}"
+            data-navigation="$ctrl.model.params.selectedOffer"
+            data-on-cancel="$ctrl.resumeOrder()"
+            data-editable="!$ctrl.isLoading"
+            data-valid="$ctrl.model.params.selectedCountry || $ctrl.isPrivateCloudOffer"
+            data-submit-text="{{:: 'ip_agora_table_submit_text' | translate }}"
+            data-cancel-text="{{:: 'ip_agora_table_cancel_text' | translate}}"
+            data-prevent-next
+        >
+            <div data-ng-show="$ctrl.loading.ipOffers" class="text-center">
+                <oui-spinner></oui-spinner>
+            </div>
+            <div class="row px-2 mb-2">
+                <div
+                    class="col-4 px-1"
+                    data-ng-show="$ctrl.isFailoverShowable()"
                 >
-                    <option
-                        value=""
-                        data-ng-disabled="true"
-                        data-translate="global_select"
+                    <oui-select-picker
+                        class="text-center"
+                        data-ng-if="!$ctrl.loading.ipOffers"
+                        data-ng-disabled="!$ctrl.failoverIpOffers.length"
+                        data-name="productName"
+                        data-match="productDisplayName"
+                        data-placeholder="{{:: 'ipv4_select_placeholder' | translate }}"
+                        data-model="$ctrl.model.params.selectedOffer"
+                        data-values="$ctrl.failoverIpOffers"
+                        data-on-change="$ctrl.onSelectedOfferChange('IPv4Failover')"
+                        data-variant="secondary"
+                        data-required
                     >
-                    </option>
-                </select>
-                <select
-                    id="offer"
-                    class="form-control"
-                    data-ng-if="ctrl.isPrivateCloudOffer"
-                    data-ng-model="ctrl.model.params.selectedOffer"
-                    data-ng-change="ctrl.onSelectedOfferChange()"
-                    data-ng-disabled="!ctrl.ipOffers.length"
-                    data-ng-options="offer as (ctrl.getOfferContent(offer)) for offer in ctrl.ipOffers | orderBy:'price.value' track by offer.planCode+offer.maximumQuantity"
+                        <oui-select-picker-description>
+                            <strong>
+                                <h4
+                                    class="text-center"
+                                    data-translate="ip_agora_failover_ip_title"
+                                ></h4>
+                                <p
+                                    class="text-center"
+                                    data-translate="ipv4Failover_price_label"
+                                ></p>
+                            </strong>
```

Comment:
**nitpick:** Remove usage of block elements (`<h4>`) in an inline elements(`<strong>`)